### PR TITLE
Modified LTW tests with oneAuthTestApp to run with MSA accounts

### PR DESCRIPTION
**Pull request by somalaya**
(Same as title)
Related PR : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2244

Also removed 2 tests that are not required anymore.

### CODE REVIEW

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/BrokerTestHelper.java [hunk 1]
```diff
@@ -44,6 +44,8 @@ public static ITestBroker createBrokerFromFlavor(@Nullable final SupportedBroker
                 return new BrokerMicrosoftAuthenticator();
             case BuildConfig.BrokerCompanyPortal:
                 return new BrokerCompanyPortal();
+            case BuildConfig.BrokerLTW:
+                return new BrokerLTW();
```
Comment (fadidurah) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/BrokerTestHelper.java:48]: AutoBroker would already cover this no? Probably good to add anyway

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571508.java [hunk 2]
```diff
@@ -42,14 +44,35 @@
...
+
+    public TestCase2571508(@NonNull UserType userType) {
+        mUserType = userType;
+    }
+
+    @Parameterized.Parameters(name = "{0}")
+    public static List<UserType> userType() {
+        return Arrays.asList(
+                UserType.MSA,
+                UserType.CLOUD
```
Comment (fadidurah) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571508.java:72]: You are not using these for the lab account query, that needs to be added.

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571565.java [hunk 3]
```diff
@@ -119,14 +142,17 @@ public void test_2571565() throws Throwable {
 
     @Override
     public LabQuery getLabQuery() {
-        return null;
+        return LabQuery.builder()
+                .userType(UserType.CLOUD)
```
Comment (fadidurah) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571565.java:146]: Hardcoding Cloud type?

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571580.java [hunk 4]
```diff
@@ -103,12 +125,14 @@ public void test_2571580() throws Throwable {
 
     @Override
     public LabQuery getLabQuery() {
-        return null;
+        return LabQuery.builder()
+                .userType(UserType.CLOUD)
```
Comment (fadidurah) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2571580.java:129]: Same as above

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2582290.java [hunk 5]
```diff
@@ -147,14 +168,17 @@ public void test_2582290() throws Throwable{
 
     @Override
     public LabQuery getLabQuery() {
-        return null;
+        return LabQuery.builder()
+                .userType(UserType.CLOUD)
```
Comment (fadidurah) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2582290.java:172]: Same as above

**Review by fadidurah — APPROVED** [review]

**Review by rpdome — APPROVED** [review]

**Pull request merged**